packages/jest-editor-support/src/Runner.js

```diff
@@ -17,16 +17,21 @@ const {EventEmitter} = require('events');
 const ProjectWorkspace = require('./ProjectWorkspace');
 const {jestChildProcessWithArgs} = require('./Process');
 
+import type {Options} from './types';
 // This class represents the running process, and
 // passes out events when it understands what data is being
 // pass sent out of the process
 module.exports = class Runner extends EventEmitter {
   debugprocess: ChildProcess;
-  workspace: ProjectWorkspace;
+  options: Options;
   outputPath: string;
+  workspace: ProjectWorkspace;
 
-  constructor(workspace: ProjectWorkspace) {
+  constructor(workspace: ProjectWorkspace, options?: Options) {
     super();
+    this.options = options || {
+      createProcess: jestChildProcessWithArgs,
+    };
```

Comment:
can you change this to:

```
this.options = {
  createProcess: (options && options.createProcess) || jestChildProcessWithArgs,
};
```

This will allow future options to work properly.

While you are at it, you could rename the function to "createJestProcessWithArgs" or something. The current name is weird.